.github/workflows/test_rust.yml

```diff
@@ -99,7 +99,7 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: swf_images
-          path: tests*/**/actual.png
+          path: tests*/**/actual*.png
```

Comment:
BTW Why `tests*/` and not `tests/`?

Reply:
This is a workaround to preserve the directory structure in the uploaded archive: https://github.com/actions/upload-artifact/issues/206#issuecomment-1157428735